libraries/rush-lib/src/logic/pnpm/PnpmOptionsConfiguration.ts

```diff
@@ -32,6 +32,12 @@ export type PnpmStoreOptions = PnpmStoreLocation;
  */
 export type PnpmResolutionMode = 'highest' | 'time-based' | 'lowest-direct';
 
+/**
+ * Possible values for the `pnpmLockfilePolicies` setting in Rush's pnpm-config.json file.
+ * @public
+ */
+export type PnpmLockfilePolicy = 'disallowInsecureSha1';
```

Comment:
"disallowInsecureSha1" implies that there are secure SHA1 hashes.

Maybe call this just "disallowSha1"? @octogonz?

Reply:
I thought that "insecure" would more clearly explain why someone would want to disallow it